### PR TITLE
Update actioneffect.py

### PR DESCRIPTION
**Pull request by rexatlanti**
added RawEmnity

### CODE REVIEW

File: nari/types/actioneffect.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class EffectType(IntEnum):
     ApplyStatusEffectTarget = 15
     ApplyStatusEffectSource = 16 # effect entry on target but buff applies to source, like storm's eye
     StatusNoEffect = 20
+    RawEnmity = 24 # used by provoke. ActionEffect.value equals the amount of enmity gained
```
Comment (nonowazu) [inline nari/types/actioneffect.py:26]: Thank you for adding this! Just one small change – pylint will want docstrings on this, would you mind editing that comment into a makeshift docstring?

```python
    RawEnmity = 24
    """Used by provoke. ActionEffect.value equals the amount of enmity gained"""
```

**Review by nonowazu — CHANGES_REQUESTED** [review]
Thank you very much for contributing! Just one very tiny change and I can accept this.

File: nari/types/actioneffect.py (hunk 1 above)
Comment (rexatlanti) [reply nari/types/actioneffect.py]: did the thing \o/

**Review by nonowazu — APPROVED** [review]
Thank you!

**Pull request merged**